src/cgame/cg_buildable.cpp

```diff
@@ -739,7 +739,7 @@ void CG_InitBuildables()
 		trap_UpdateScreen();
 	}
 
-	cgs.media.teslaZapTS = CG_RegisterTrailSystem( "teslaZapTS" );
+	cgs.media.teslaZapTS = CG_RegisterTrailSystem( "trails/weapons/tesla/lightning" );
```

Comment:
This is actually ReactorLighting. The tesla was removed and this is only used by the RC now.

Reply:
Its also used by the adv mara, by the way it's loaded as another trail :p
I thought about that and I will rename it, I prefer to do it in two steps but it can just be another commit in the same PR.